Source/VCGeneration/Split.cs

```diff
@@ -1412,6 +1412,11 @@ public void ReadOutcome(ref ConditionGeneration.Outcome curOutcome, out bool pro
             checker.ProverRunTime.TotalSeconds, outcome);
         }
 
+        if (CommandLineOptions.Clo.XmlSink != null) {
```

Comment:
How come you don't need ` && splitNum >= 0` like in the if condition above?

Reply:
The `WriteEndSplit` method doesn't actually refer to `splitNum` (since that's already recorded in the parent `<split ...>` element), unlike the tracing output. I have a slight preference for still recording the data even if something weird happens and `splitNum < 0`, but I don't feel too strongly and could add the condition if you'd prefer.

Reply:
Got it. Makes sense.